test/MediatR.Tests/SendTests.cs

```diff
@@ -213,4 +251,59 @@ public async Task Should_raise_execption_on_null_request()
 
         await Should.ThrowAsync<ArgumentNullException>(async () => await mediator.Send(default!));
     }
+
+    [Fact]
```

Comment:
Nitpick, but these tests don't exercise those service scanning tests. You'd want to use `services.AddMediatR` instead. Which I am fine with, I need to convert all the tests to stop using Lamar anyway.

Reply:
Ok Mr. @jbogard , I believe the updates to the pr have done what you suggest.  I converted all of the tests in this class to use the default service provider instead of Lamar.  If you'd like I can convert tests in other classes too.. I just didn't want to change too much so that the PR isn't too big.  Also might be a bit out of scope for what this pr is meant for.  But if you'd like I can create a new PR to remove the dependency on Lamar completely for other test classes as well.

Reply:
Ah no just these tests is fine! I can circle back to the others. Thanks!